### PR TITLE
Button "view game" not working on the Game page

### LINKED ISSUE #248: Button "view game" not working on the Game page

### PR DESCRIPTION
**Pull request by Klimonov**

### CODE REVIEW

File: pages/game.vue [hunk 1]
```diff
@@ -76,34 +76,41 @@ export default {
...
-      } else {
-        this.isGameAvailable = false
-        this.$ga.event({ eventCategory: 'game', eventAction: 'notfound', eventValue: this.gameId })
-      }
-    })
+    this.fetchGame()
   },
   methods: {
+    fetchGame() {
+      import('../static/js/anthive-5.0.js').then(() => {
```
Comment (kezlya) [inline pages/game.vue]: Let's Use player name from `ENV` as well SIM_VERSION

**Review by kezlya — COMMENTED** [review]
Looks Good but while we are here let's remove hardcode from player and use ENV

**Review by EgoriusRus — APPROVED** [review]
LGTM

**Review by kezlya — APPROVED** [review]
LGTM

**Pull request merged**